Shoko.Server/Providers/AniDB/HTTP/RequestGetAnime.cs

```diff
@@ -13,21 +13,19 @@ public class RequestGetAnime : HttpRequest<ResponseGetAnime>
     private readonly HttpXmlUtils _xmlUtils;
     private readonly HttpAnimeParser _parser;
     private readonly string _aniDBUrl;
-    private readonly ushort _aniDBPort;
 
     public int AnimeID { get; set; }
 
     protected override string BaseCommand =>
-        $"http://{_aniDBUrl}:{_aniDBPort}/httpapi?client=animeplugin&clientver=1&protover=1&request=anime&aid={AnimeID}";
+        $"{_aniDBUrl}/httpapi?client=animeplugin&clientver=1&protover=1&request=anime&aid={AnimeID}";
```

Comment:
Instead of us needing to set the URL on every single one, would it be worth setting this? https://learn.microsoft.com/en-us/dotnet/api/system.net.http.httpclient.baseaddress?view=net-8.0

Reply:
I'll go cry in a corner wondering why I didn't think of that 😔